xapian-applications/omega/query.cc

```diff
@@ -1940,6 +1955,19 @@ eval(const string &fmt, const vector<string> &param)
 		value = args[0];
 		url_prettify(value);
 		break;
+	    case CMD_qid: {
+		string qstr = args[0];
+		trim_string(qstr);
```

Comment:
What's the reason for removing leading and trailing spaces (rather than just processing the query as-is, or performing more normalisation)?